modules/renter/hostdb/hostweight.go

```diff
@@ -8,16 +8,22 @@ import (
 
 var (
 	// Because most weights would otherwise be fractional, we set the base
-	// weight to 10^80 to give ourselves lots of precision when determing the
+	// weight to 10^150 to give ourselves lots of precision when determing the
 	// weight of a host
 	baseWeight = types.NewCurrency(new(big.Int).Exp(big.NewInt(10), big.NewInt(150), nil))
 )
 
 // calculateHostWeight returns the weight of a host according to the settings of
 // the host database entry. Currently, only the price is considered.
 func calculateHostWeight(entry hostEntry) (weight types.Currency) {
+	// In the weighted price, the download bandwidth price is multiplied by
+	// numDownloads, which is our guess at the median number of downloads per
+	// file (some files may go untouched, but others may be requested
+	// many times).
+	const numDownloads = uint64(50)
```

Comment:
`consts` never need explicit types.\* Not a big deal though.

\*I'm not 100% sure this is true, but the only counterexample I could think of was:
```go
type foo int

func (f foo) times2() foo {
    return f*2
}

func main() {
	const f foo = 3
	println(f.times2())
}
```
Even here, you could typecast at the call site (and I believe this incurs no overhead):
```go
func main() {
	const f = 3
	println(foo(f).times2())
}
```